docs/systemd.md

```diff
@@ -0,0 +1,43 @@
+## Running with systemd
+
+Once you've verified that you can run launcher in your shell, you'll likely want to keep launcer running in the background and after the endpoint reboots. To do that we recommend using [systemd](https://coreos.com/os/docs/latest/getting-started-with-systemd.html)
+
+Below is a sample unit file.
+
+```
+[Unit]
+ Description=The Kolide Launcher
+ After=network.service syslog.service
+
+ [Service]
+ ExecStart= $LauncherPath \
+ --hostname=$FleetServer:FleetPort \
+ --enroll_secret=$FleetSecret \
+ --insecure \
```

Comment:
I would advise not including this flag as a default suggestion. You can document it, just dont make it copy/pasted.

Reply:
Completely agree. Added a line informing this flag as optional and only for test purposes.